package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "kvue",
-	"version": "2.508.0",
```

Comment:
Updating the vue version can introduce issues so we probably shouldn't do that unless it's part of a bigger effort where we have a test plan in place and everyone is made aware of the change.

Reply:
This one can sometimes happen but I'm more concerned that there is some much other work here.

My suggestion is to change this PR to be into `main` instead of a feature branch. This is a small segment of work that should be ok to work against main on.

Reply:
This is the project build version. It can still be reverted though.

Reply:
@mcstover Yeah looks like the target branch needs to get `main` merged in first, I added a separate comment.